frontend/express/public/javascripts/countly/countly.common.js

```diff
@@ -4091,9 +4091,11 @@
         * Parse second to standard time format
         * @memberof countlyCommon
         * @param {number} second  number
-        * @returns {string} return format "HH:MM:SS"
+        * @param {number} [trimTo=3]  number [1,3]
+        * @returns {string} return format "HH:MM:SS", if trimTo is specified the length of the result is trimmed
+        * @example trimTo = 2, HH:MM:SS result will be trimmed to HH:MM
         */
-        countlyCommon.formatSecond = function(second) {
+        countlyCommon.formatSecond = function(second, trimTo = 3) {
```

Comment:
I think using default as 3 kind of breaks the default behavior and other places that expect full output would not get it anymore?

Reply:
No, the default 3 will make the function behave as it was doing before the trimTo argument was added.

Reply:
hmm, correct me if I am wrong, but by default, there are possible 5 tiers (year, day, hour, minute, second) so if there are some cases where it expects to return all 5, it would not only get 3 and thus break old behavior?

Reply:
yes you are correct, the default should have been 5, I did not consider YY:DD:HH:MM:SS, I will rectify it